utilities/ride-model-util/src/main/java/com/adobe/ride/utilities/model/ModelObject.java

```diff
@@ -278,6 +278,37 @@ private void initializeModelObject(String resourceLocation, JSONObject presetNod
     loadModel(resourceLocation);
   }
 
+  /**
+   * Create a model object from a json schema string and use a predefined set of nodes to start
+   * with. Also only build a targeted set of nodes. The specified resource location is only used as
+   * metadata, so the content will not be fetched from it.
```

Comment:
Can we tighten up this description a little?  Maybe something like: "Static constructor to initialize with a JSON schema string.  Optional methods for adding preset nodes, defining target nodes, and whether to use only required nodes."

Reply:
Sure!